lib/components/artist/articles.js

```diff
@@ -41,3 +38,22 @@ const styles = StyleSheet.create({
     fontSize: 20,
   }
 });
+
+export default Relay.createContainer(Articles, {
+  fragments: {
+    artist: () => Relay.QL`
+      fragment on Artist {
+        name
+        articles {
```

Comment:
You should move this as a `article` fragment to the `Article` component. Each component should only specify attributes that it uses _itself_.